docs/source/python_sdk.rst

```diff
@@ -47,17 +33,61 @@ Helper
 
 .. autofunction:: globus_automate_client.create_flows_client
 
-Example
--------
+Examples
+^^^^^^^^
+
+SDK Helpers
+-----------
+
+If you're running the SDK locally in a secure location, you will probably want
+to use the provided convenience functions, ``create_action_client`` and
+``create_flows_client``. These functions share functionality with the
+``globus-automate`` CLI to handle loading and storing Globus Auth
+tokens on the local filesystem and trigger interactive logins when additional
+consents are required to interact with Actions or Flows:
+
+.. literalinclude:: code_snippets/create_action_client.py
```

Comment:
This is a nice way to do this so the code is also stand-alone in a file for easier browsing and inclusion in other projects etc.